src/config_dialog.c

```diff
@@ -149,8 +150,9 @@ struct {
     {"save-as"          , save_as           },
     {"manual"           , manual            },
     {"syntax"           , syntax            },
-    {"read-only"        , read_only_cmd     },
+    {"read-only"        , read_only         },
```

Comment:
Isn't there any problem having a variable with the same name of a function?

Reply:
> Isn't there any problem having a variable with the same name of a function?

I removed the read_only global variable and put it as a member of the struct BUFFER in a previous commit

Reply:
I forgot that.